add page: auvaltool (#8689)
tldr-pages/tldr: pages/osx/auvaltool.md

# auvaltool

> > Mac OS X AudioUnit validation tool.

Reviewer: ```suggestion
> AudioUnit validation tool for Mac.
```
I think mentioning "Mac" without Specifying OS X would be better,

> > More information: <https://www.unix.com/man-page/mojave/1/auvaltool>.

Reviewer 2: We commonly use https://ss64.com/osx/ for this, but since the manpage isn't mentioned on the site, I think it's ok to leave this URL as-is.

What do you thing @pixelcmtd?

Reviewer: Recently, there was a discussion regarding this in #8670 ([Link](https://github.com/tldr-pages/tldr/pull/8670#issuecomment-1264892949)). There isn't a proper manpage (website) with support with all commands for mac. I think the current link is fine so let's see what pixelcmtd thinks.

Reviewer 3: @reinhart1010 https://duckduckgo.com/?q=site%3Ass64.com+auvaltool
We could use https://keith.github.io/xcode-man-pages/auvaltool.1.html instead, but I'd just leave it as is.

> - List all available AudioUnits of any type:

Reviewer: ```suggestion
- List all [a]vailable AudioUnits of any type:
```

`auvaltool -a`

> - List all available AudioUnits of any type and their location:

Reviewer: ```suggestion
- List all [a]vailable AudioUnits of any type with their [l]ocation:
```
I think this would be better.
